ELLI/src/app/features/signup/signup.component.html

```diff
@@ -11,13 +11,12 @@ <h2 class="center">Signup</h2>
                 <label for="password">Password</label>
                 <input type="password" name="password" id="password" placeholder="Password">
             </div>
-            <div class="form-element">
-                <label for="password-conf">Password</label>
-                <input type="password" name="password-conf" id="password-conf" placeholder="Confirm Password">
-            </div>
         </form>
         <div class="btn-container">
             <button type="button" id="submit-btn">Submit</button>
         </div>
     </div>
-</section>
\ No newline at end of file
+</section>
+    <!--The link to connect an html and css file--> 
+    <link rel="stylesheet" href="signup.component.css">
```

Comment:
delete this link. the linking is handled in the .ts file, look at line 8 of signup.component.ts

Reply:
watching that.. i believe those changes were made by jared will confirm with him one more time before making the change.